results-tabulation-api/api/TallySheetVersionApi/__init__.py

```diff
@@ -168,9 +168,10 @@ def create(tallySheetId, body):
     tally_sheet = TallySheet.get_by_id(tallySheetId=tallySheetId)
 
     # validate user inputs to prevent XSS attacks
-    if not input_is_valid(request_body.get("content")):
-        raise InvalidInputException(
-            message="Invalid input detected. Use of disallowed characters/invalid input length detected",
+    input_is_valid, error_message = validate_tally_sheet_version_request_content_special_characters(request_body.get("content"))
+    if not input_is_valid:
```

Comment:
Move the exception to the function itself.

Reply:
done